src/main/java/skyler/main/Storage.java

```diff
@@ -0,0 +1,107 @@
+package skyler.main;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import skyler.task.Deadline;
+import skyler.task.Event;
+import skyler.task.Task;
+import skyler.task.ToDo;
+import skyler.exception.SkylerException;
+
+public class Storage {
+    private static final String FILE_PATH = "./data/Skyler.txt";
+    private static List<Task> tasks = new ArrayList<>();
+
+    /**
+     * Loads tasks from the specified file path and adds them to the common tasks
+     * list.
+     * Assumes the file format is compatible with the task parsing logic in
+     * parseTaskFromFile method.
+     */
+    public static void loadTasksFromFile() {
+        File file = new File(FILE_PATH);
+        try {
+            Scanner scanner = new Scanner(file);
+            while (scanner.hasNext()) {
+                String data = scanner.nextLine();
+                Task task = parseTaskFromFile(data);
+                tasks.add(task); // Add the task to the common list
+            }
+            scanner.close();
+        } catch (IOException | SkylerException e) {
+            System.out.println("An error occurred while loading tasks.");
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Parses a task from the given data string, determining the task type (ToDo,
+     * Deadline, Event)
+     * and creating the corresponding Task object.
+     *
+     * @param data The string containing the task data in a specific format.
+     * @return The Task object parsed from the data string.
+     * @throws SkylerException If there is an error parsing the task or if the task
+     *                         type is unknown.
+     */
+    public static Task parseTaskFromFile(String data) throws SkylerException {
+        String taskType = data.substring(1, 2); // Extracting task type (T, D, E)
+        boolean isDone = data.charAt(4) == 'X'; // Assuming 'x' represents a completed task
+        String details = data.substring(7).trim(); // Extracting task details
+
+        switch (taskType) {
+            case "T":
```

Comment:
Note the switch statement coding style defined in the [given Java coding standard](https://se-education.org/guides/conventions/java/intermediate.html#layout). The case and default clauses shouldn't be indented. You can configure IntelliJ or your IDE to handle this automatically.